src/example-types/filters/tagsQuery.js

```diff
@@ -81,16 +81,17 @@ let filter = ({ tags, join, field, exact }) => ({
   },
 })
 
-let buildResultQuery = node => ({
+let buildResultQuery = (node, children = {}, groupsKey = 'tags') => ({
```

Comment:
These changes allow the query to be composed in the pivot type

Reply:
Some more info:

- `children` does the actual composition of child aggregations
- `groupsKey` is a thing the pivot type uses to handle groups vs columns